src/Features/Auth/epics.js

```diff
@@ -108,9 +110,11 @@ export const logoutEpic = (action$, state) =>
         body: refreshToken,
       }).pipe(
         mergeMap(() => {
+          History.push('/login');
           return of(logoutSuccess());
         }),
         catchError(() => {
+          History.push('/login');
```

Comment:
Is history.push working fine here as we have discovered that history from createBrowserHistory change the URL but not rendering the appropriate component ??

Reply:
outdated code